docs/index.rst

```diff
@@ -26,10 +26,10 @@ Web
 The easiest way to get started with tracing is to instrument your web server.
 We support many `Web Frameworks`_. Install the middleware for yours.
 
-Databases
-~~~~~~~~~
+Instrument modules
```

Comment:
I don't love this title since everything is a library. Maybe just leave it at databases?